src/components/DatepickerDay/index.js

```diff
@@ -44,6 +45,7 @@ const DatepickerDay = ({
         invalid={invalid}
         valid={valid}
         autoFocus={stateHasFocus}
+        borderColor={datepickerColor && datepickerColor}
```

Comment:
Is this prop being used? If so please can you add a default color (same as you did with RangePIcker)